.dockerignore

```diff
@@ -1,7 +1,5 @@
 *
 !backend
-!configs
-!environments-and-requirements
 !frontend
 !installer
```

Comment:
i think this needs to be updated to `binary_installer`

Reply:
oh good catch. fixing.